docs/ref/csrf.txt

```diff
@@ -92,6 +92,13 @@ protection for your views as outlined above.
     The CSRF token cookie is named ``csrftoken`` by default, but you can control
     the cookie name via the :setting:`CSRF_COOKIE_NAME` setting.
 
+    The CSRF header name is ``HTTP_X_CSRFTOKEN`` by default, but you can control
+    the header name via the :setting:`CSRF_HEADER_NAME` setting.
+
+    This header like any other http headers, before it appears in ``request.META``
```

Comment:
Hmm I confused the CSRF header is checked by server in CSRF middleware, and the value of that header is taken from request.META. Can you elaborate on that?

For controlling CSRF cookie (name) indeed we use CSRF_COOKIE_NAME.